### DISCUSSION TITLE
Merge with the Development

### DISCUSSION THREAD

**Discussion by PranishKansakar** [PranishKansakar/Tafesa240154#11] [Announcements]
Hello guys just a reminder that we need to merge the feature branch with the development branch by this week. 

I have already merged currency calculator feature to the development. 

You may have conflict on the merge as I have added a exit() code in the menu page. If you have any problem do let me know. 

Pranish

**Comment by Omonotuaga** [comment]
Hey Pranish,

Thanks for the heads-up! We appreciate you taking care of the merge and letting us know about the potential conflict with the exit() code. We'll keep an eye out for any issues and reach out if we need assistance.

Cheers,
Tj

**Comment by GEONHWISONG** [comment]
Thank you for the reminding! i will make it be fore this week!
